tests/parsers/locate.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for locate database file parser."""
+
+import unittest
+
+from plaso.parsers import locate
+
+from tests.parsers import test_lib
+
+
+class LocateUnitTest(test_lib.ParserTestCase):
+  """Tests for Locate Database file parser."""
+
+  def testParseFile(self):
+    """Test parsing of a Locate Database file."""
+    parser = locate.LocateDatabaseParser()
+    storage_writer = self._ParseFile(['mlocate.db'], parser)
+
+    self.assertEqual(storage_writer.number_of_events, 6)
+    self.assertEqual(storage_writer.number_of_extraction_warnings, 0)
+    self.assertEqual(storage_writer.number_of_recovery_warnings, 0)
+
+    events = list(storage_writer.GetEvents())
+
+    expected_event_values = {'folder_path': '/home/user/temp'}
```

Comment:
Check additional event values like date_time

Reply:
Updated tests to also check date_time of event